azure-pipelines.release.yml

```diff
@@ -11,10 +11,11 @@ variables:
 pool: 'Self Host Ubuntu'
 
 schedules:
-  # minute 0, hour 12 in UTC (5am in UTC+7), any day of month, any month, days 1-5 of week (M-F)
+  # minute 0, hour 7 in UTC (11pm in UTC-8), any day of month, any month, days 1-5 of week (M-F)
```

Comment:
If 11pm doesn't help or ends up making things worse, we could try something different. It's just hard to find a time that doesn't cause conflicts with either Redmond or Europe (especially when work often goes into the evening)...